### DISCUSSION TITLE
Gas limit

### DISCUSSION THREAD

**Discussion by Osuolale1** [Cyfrin/foundry-full-course-cu#442] [Q&A]
Good day great minds

Having encounter with gas limit
![IMG-20230725-WA0010](https://github.com/Cyfrin/foundry-full-course-f23/assets/101529913/b49a642b-a52c-4171-8ec3-20ffa86c2aeb)

**Comment by PatrickAlphaC** [comment]
Hello! Thanks for making this issue. 

Could you please remember to follow this section to format your questions? https://youtu.be/umepbfKp5rI?t=22464

We'd need more information in order to triage this issue.
